src/main/java/io/appium/java_client/remote/AppiumCommandExecutor.java

```diff
@@ -53,11 +53,12 @@
 import static java.util.Optional.ofNullable;
 import static org.openqa.selenium.remote.DriverCommand.NEW_SESSION;
 
+@NullMarked
 public class AppiumCommandExecutor extends HttpCommandExecutor {
 
     private final Optional<DriverService> serviceOptional;
     @Getter
-    private final HttpClient.Factory httpClientFactory;
+    private final Factory httpClientFactory;
```

Comment:
This is actually wrong. We should be using the original field from the parent class instead of duplicating it here same way we fetch the property value for getAdditionalCommands